app/controllers/order_option_assignments_controller.rb

```diff
@@ -43,24 +44,19 @@ def edit
       end
 
       empty_assignment_cnt = 0
-      @collections_to_list.each do |collection|
-        if collection['option-def'].nil?
-          empty_assignment_cnt += 1
-        end
+      @collections_to_list.each do |coll|
```

Comment:
minor, but I thought you wanted to avoid abbreviated variable/parameter names that can be ambiguous.